python/ray/rllib/utils/annotations.py

```diff
@@ -27,8 +27,8 @@ def PublicAPI(obj):
     can expect these APIs to remain stable across RLlib releases.
 
     Subclasses that inherit from a ``@PublicAPI`` base class can be
-    assumed part of the RLlib public API as well (e.g., all agent classes
-    are in public API because Agent is ``@PublicAPI``).
+    assumed part of the RLlib public API as well (e.g., all trainer classes
+    are in public API because Trainer is ``@PublicAPI``).
 
     In addition, you can assume all agent configurations are part of their
```

Comment:
`trainer configurations` -- I'm not going to look too carefully in the codebase for missed parts but maybe go through another pass?

Reply:
Done